app/App.js

```diff
@@ -74,19 +74,21 @@ export default {
   },
   template: `
     <Header :title="metadataConfig.header_title"></Header>
-    <div class="container">
+    <main role="main">
```

Comment:
```suggestion
    <main>
```

The main element implicitly has the main role already, so [mdn recommends using the tag instead of the role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/main_role#best_practices).

Reply:
Reading the documentation, I wasn't sure if we should add an `aria-labelledby` or if it will just automatically use the first `h1` element?

Reply:
Trying it out on voiceover, in the Rotor landmark list:

- Without aria-labelledby, it just says "main"
- With aria-labelledby, it says "Browse Sales Catalogs main" (or whatever the text of the element you are using as the label).

I think just "main" gets the point across more succinctly that this is the main content of the page, so I think it's great in its current state.